TCAT/Supporting Files/Constants.swift

```diff
@@ -54,22 +102,90 @@ struct Constants {
         static let informationCellIdentifier = "InformationCell"
     }
 
+    /// The empty state messages
+    struct EmptyStateMessages {
+        // Error messages
+        static let couldntGetStops = "Couldn't Get Stops"
+        static let locationNotFound = "Location Not Found"
+        static let noNetworkConnection = "No Network Connection"
+        static let noRoutesFound = "No Routes Found"
+
+        // Other empty state messages
+        static let lookingForRoutes = "Looking For Routes..."
+    }
+
     struct Footers {
         static let emptyFooterView = "RouteDetailEmptyFooterView"
         static let phraseLabelFooterView = "RouteDetailPhraseLabelFooterView"
     }
 
+    /// The phrases used in InformationViewController
+    struct InformationView {
+        static let onboarding = "Show Onboarding"
+        static let sendFeedback = "Send Feedback"
+        static let moreApps = "More Apps"
+        static let website = "Visit Our Website"
+        static let madeBy = "Made by Cornell App Development"
+        static let appDevDescription = "An Engineering Project Team\nat Cornell University"
```

Comment:
Not ideal to be on this page. Any thoughts on how we could do this? Two labels maybe